src/qml/pages/main.qml

```diff
@@ -22,18 +23,25 @@ ApplicationWindow {
         anchors.fill: parent
     }
 
-    Wizard {
+    Page {
```

Comment:
Is the containing Page necessary? Can this Item just be the swipe view?

Reply:
Actually this should be defined as a Component so it isn't being loaded twice by the StackView and the ApplicationWindow. I think it's been working simple because the first one is hiding the second by rendering over it.

Reply:
You'll need the background and anchor properties removed when changing to to a Component as well.